abel/tools/basis.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
```

Comment:
This seems like a good idea to use a single function to deal with the loading and saving of basis sets. Is it possible for all of the basis-set-requiring transform functions (basex, three-point) to use this? You can perhaps open a new issue that we need to do this.